### PR TITLE
Quota: fix the note on quota for setting the limit

### PR DESCRIPTION
**Pull request by harigowtham**
Signed-off-by: hari gowtham <hgowtham@redhat.com>

### CODE REVIEW

**Comment by harigowtham** [comment]
@dolanoi @prashanthpai @amarts @humblec @atinmu Please do take a look when free.
The issue was brought to light by an upstream user.

**Comment by dolanoi** [comment]
Looks good to me.

File: docs/Administrator Guide/Directory Quota.md [hunk 1]
```diff
@@ -21,8 +21,9 @@ You can set the quota at the following levels:
 -   **Volume level** – limits the usage at the volume level
 
 > **Note**
-> You can set the disk limit on the directory even if it is not created.
-> The disk limit is enforced immediately after creating that directory.
+> You can set the disk limit on the directory even if files are not created.
```
Comment (prashanthpai) [inline docs/Administrator Guide/Directory Quota.md]: `You can set the quota limit on an empty directory. The quota limit will be automatically enforced when files are added to the directory.`

Comment (harigowtham) [reply docs/Administrator Guide/Directory Quota.md]: done

**Pull request merged**